README.md

```diff
@@ -49,6 +49,29 @@ profile_metrics_alerting::tools::whitelabel_subtitle: "Managed by ORGANIZATION G
 profile_metrics_alerting::tools::whitelabel_title: "Monitoring and Telemetry Interface"
 ```
 
+To pause/start alerts on a schedule (applicable for old style alerts only), you can use these parameters:
+```yaml
+# Format for command is similar to running alert_toggle.sh by hand, see README in files/root/grafana_tools
+profile_metrics_alerting::alert_cycle::crons:
+   "Pause mforgehn1 alerts":
+    command: "pause tag \"mforgehn1\""
+    hour: 21
+    minute: 55
+    weekday: 0
+  "Start mforgehn1 alerts":
+    command: "start tag \"mforgehn1\""
+    hour: 0
+    minute: 45
+    weekday: 1
+profile_metrics_alerting::alert_cycle::cycle_alerts: true
+```
+You'll want to define your crons for `profile_metrics_alerting::alert_cycle::crons` in any role that uses this repo (For example if you have a primary/secondary role). This is because if you swap roles you want the alert pausing/starting crons to be removed from the secondary role so you don't get alerts enabled in multiple places leading to duplicate alerts. 
+
+### class profile_metrics_alerting::ssh (
+-  Array[String] $metrics_node_ips,
+-  String $sshkey_pub,
+-  String $sshkey_priv,
+-  String $sshkey_type,
```

Comment:
Should all of these parameters (including the others not tagged by this comment) be moved lower into the README file under the `Reference` heading?